Story/Episode 5/st_056070.csv

```diff
@@ -1,25 +1,25 @@
-st_056070_0170#0,"""I've heard that Cuent's Guardian came from another world."""
-st_056070_0180#0,"""With your power Margareta, I'm sure that you can do it."""
-st_056070_0190#0,"""Huh, that was interesting..."""
-st_056070_0200#0,"""Oh, that wasn't a bad idea. I guess that boring guy was right after all."""
-st_056070_0210#0,"""What a waste... Let's start the preparation immediately."""
+st_056070_0010#0,"""To the west of Cuent, lay the Holy<br>Kingdom of Epic."""
+st_056070_0020#0,"""A kingdom residing within a harsh desert<br>dedicated to worshipping their<br>singular all-powerful God."""
+st_056070_0030#0,"""It is here that the Divine Queen of Epic, Margareta,<br>listened with indifference to a report on the<br>continuing troubles of the neighboring Cuent."""
+st_056070_0040#0,"""...thus concludes the report on the situation, and Cuent's repute on the continent continues to grow."""
+st_056070_0050#0,"""Yeah, yeah, I get it. Basically all you said was, 'Cuent beat Verun and ruined their rebellion' right?"""
+st_056070_0060#0,"""As quick on the uptake as ever, I see. It is just as you say, Your Holy Majesty."""
+st_056070_0070#0,"""There is an additional matter though. They have asked all the neighboring countries to sign a treaty committing to the eradication of Ephimera..."""
+st_056070_0080#0,"""Ephimera? Ah yeah, that creepy flower. I couldn't care less about it."""
+st_056070_0090#0,"""Ugh, this really sucks though."""
+st_056070_0100#0,"""What exactly 'sucks', may I ask?"""
+st_056070_0110#0,"""It's not fair! Cuent has everything... They've been around longer sure, but they have both their Sword Maiden and that new Guardian who just arrived."""
+st_056070_0120#0,"""You'd think they'd at least be able to share something cool! There's nothing but desert here."""
+st_056070_0130#0,"""Here it's always just 'God this' and 'God that' and 'God will sort it out'. Everything just gets handwaved away like that."""
+st_056070_0140#0,"""We need entertainment! I could have my own Guardian, and we could weave a tale of saving the kingdom from crisis! It'd be a refreshing change of pace."""
+st_056070_0150#0,"""Ha ha, just as you say. In that case, shall I summon forth a Guardian for you, my Queen?"""
+st_056070_0160#0,"""Summon... We can do that?"""
+st_056070_0170#0,"""I've heard tale that Cuent's Guardian hails from another world."""
+st_056070_0180#0,"""And with your power Margareta, I'm positive that we can summon someone who meets your expectations."""
+st_056070_0190#0,"""Hmm... That certainly sounds like fun."""
+st_056070_0200#0,"""You sure come up with pretty good ideas. I guess I was right after all to dispose of that stuffy old priest and replace him with you."""
+st_056070_0210#0,"""Your words are too good for me, my Queen. I shall commence the preparations immediately."""
 skit01#0,"""The Apprentices"""
-skit02#0,"""Summoning [Apprentice] in the Desert"""
+skit02#0,"""The Apprentices summoned to the desert."""
```

Comment:
Original title here was `砂漠に喚ばれし【若人】たち`, if you can see about adding [Apprentice].

Reply:
Basically it's pluralized there, so rather than write something weird like [Apprentice]s I decided to simply drop the brackets for it. It mostly just looked strange is all.